app/src/types/dbColumns.types.ts

```diff
@@ -37,7 +38,7 @@ export const typedLoggedCallModelResponse = <T extends Pick<LoggedCallModelRespo
 
 const fineTuneTestingEntrySchema = z
   .object({
-    output: chatMessage.passthrough().optional(),
+    output: chatCompletionMessage,
```

Comment:
Does this need to be optional?

Reply:
FineTuneTestingEntry.output is nullable in the database, so I think so? I actually don't remember when that field gets set relative to model creation but if we could make it non-nullable in the db then we can definitely make it non-nullable here.